Core/include/Acts/Propagator/MultiEigenStepperLoop.hpp

```diff
@@ -75,7 +76,7 @@ struct WeightedComponentReducerLoop {
   }
 
   template <typename stepper_state_t>
-  static ActsScalar momentum(const stepper_state_t& s) {
+  static ActsScalar absoluteMomentum(const stepper_state_t& s) {
     return std::accumulate(
         s.components.begin(), s.components.end(), ActsScalar{0.},
         [](const auto& sum, const auto& cmp) -> ActsScalar {
```

Comment:
I know this was like this before, but maybe it's more readable if the lambda is defined outside of `std::accumulate`?

Reply:
might be but it is consistent right now and I do not want to intrude the code too much right now